src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -132,22 +132,21 @@ def update_webapp(instance, client_affinity_enabled=None):
 
 
 def list_webapp(cmd, resource_group_name=None):
-    from ._validators import WEB_APP_TYPES
-    return _list_app(cmd.cli_ctx, WEB_APP_TYPES, resource_group_name)
+    result = _list_app(cmd.cli_ctx, resource_group_name)
+    return [r for r in result if 'function' not in r.kind] 
```

Comment:
Not very accurate now, but still good enough to filter away the `functionapp`, while still captures all the rest.
With all those unannounced backend changes on `app-kind`, it is no longer worthwhile for client sides to be accurate